en/docs/learn/try-device-flow.md

```diff
@@ -0,0 +1,84 @@
+# Try Device Authorization Grant
+
+The Device Authorization Grant is one of the grant types in the OAuth 2.0
+specification. For more information about this grant type, see
+[Device Authorization Grant](../../learn/device-flow-grant).
+
+!!! note "Before you begin" 
+    You must first set up the `Device Flow Demo App` sample webapp.
+    in order to try the following scenario.   
+    
+Step 1:
+Start WSO2 identity server by typing `sh wso2server.sh` in a terminal running on <IS_HOME>/bin/ .
+
+Step 2:
+Login to management console by providing your username and password.
+
+Step 3:
+Go to `identity->Users and Roles` and add a new user.
+![register-user](../assets/img/using-wso2-identity-server/register-user.jpg)
+
+Step 4:
+Now go to service providers and register a new service provider by providing a service provider name.
+
+![register-service-provider](../assets/img/using-wso2-identity-server/register-service-provider.png)
+
+![register-service-provider-name](../assets/img/using-wso2-identity-server/register-sp-name.jpg)
+
+Step 5:
+Go to `Service Providers-->List` and edit your service provider with OAuth.
+
+Step 6:
+Go to `Inbound Authentication Configuration-->OAuth/OpenID Connect Configuration` and click configure.
+
+![register-service-provider-oauth](../assets/img/using-wso2-identity-server/register-sp-oauth.jpg)
+
+Step 7:
+You need to tick `urn:ietf:params:oauth:grant-type:device_code` for use device flow grant type. Since these are 
+treated as public clients, always remember to put a tick on Allow authentication without the client secret. 
```

Comment:
"Since these are treated as public clients" --> we need to explain what we mean by "these". 

We can put this sentence into an info macro. E.g.,

!!! info
     Since these are public clients, ensure that the **Allow authentication without the client secret** checkbox is selected. 

Next, click **Update** to save the service provider configurations. Note the generated OAuth client key and client secret.